test/configCases/output/root-dir/test.config.js

```diff
@@ -0,0 +1,8 @@
+const path = require("path");
+const outputDirectory = __dirname.replace("configCases", "js/config");
+
+module.exports = {
+	findBundle: function() {
+		return [path.relative(outputDirectory, "/tmp/bundle.js")];
```

Comment:
If I return an absolute path here directly, the `_require` function in `ConfigTestCases.test.js` would `require` the bundle directly, making it lack of essential context utilities like the `it` function.

So the only workaround I found is to calculate the relative path of `/tmp/bundle.js`.